Models.py

```diff
@@ -0,0 +1,66 @@
+"""
+Define all data models.
+"""
+
+from py2neo.ogm import (GraphObject, Property, Related, RelatedTo, RelatedFrom)
+
+
+class Person(GraphObject):
+    __primarykey__ = "uid"
+
+    # Properties.
+    username = Property()
+    display_name = Property()
+    uid = Property()
+    email = Property()
+    photo = Property()
+
+    # Relationships.
+    Knows = Related("Person", "KNOWS")
+    Circles = RelatedTo("Circle", "PART_OF")
+    InvitedTo = Related("Event", "INVITED_TO")
+
+    def __init__(self, username, display_name, email, photo):
+        self.username = username
+        self.display_name = display_name
+        self.email = email
+        self.photo = photo
+
+class Circle(GraphObject):
+    __primarykey__ = "cid"
```

Comment:
If we leave this line out, an automatic __id__ property will be set: https://py2neo.org/v3/ogm.html#related-objects.

Also just an fyi, "uid" stands for unique identifier, so you would never really say "cid" or "eid"